lnd_test.go

```diff
@@ -5052,6 +5052,218 @@ func testMultiHopOverPrivateChannels(net *lntest.NetworkHarness, t *harnessTest)
 	closeChannelAndAssert(ctxt, t, net, carol, chanPointCarol, false)
 }
 
+func testChannelSubscription(net *lntest.NetworkHarness, t *harnessTest) {
```

Comment:
two suggestions:
1. Make the integration test change its own commit.
2. Can the same be tested by adding the channel subscription to one or more existing tests? The integration tests are long-running at this point, and if we can avoid the overhead of extra tests that would be a win I think :)

Reply:
fixed, added to `testBasicChannelCreation` and renamed the test